contrib/macdeploy/extract-osx-sdk.sh

```diff
@@ -8,7 +8,7 @@ set -e
 
 INPUTFILE="Xcode_7.3.1.dmg"
 HFSFILENAME="5.hfs"
-SDKDIR="Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk"
+SDKDIR="Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/Xcode-11.3.1-11C505-extracted-SDK-with-libcxx-headers.tar.gz"
```

Comment:
This makes no sense actually... touching this file in 16392 (which we merged recently via #4255) was also a mistake 🙈 The good news is this file is no longer used anyway, we should probably backport 20735 to avoid further confusion.

Reply:
Resolved in Part 3 as https://github.com/dashpay/dash/pull/4379/commits/334e6821a657c97c5f806f950d8d25b9c7ded8d0